src/Microsoft.Extensions.Logging.Testing/Microsoft.Extensions.Logging.Testing.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
```

Comment:
I don't really have a problem keeping this library xUnit-specific. It's not a very big library. We could split into two libraries if there was demand for it, but I don't know that I'd front-load that work now. Meh, I'm good either way.